anet-dictionary.yml

```diff
@@ -61,6 +61,7 @@ fields:
         topTaskOnceReport:
           label: Engagement assessment of objective
           recurrence: once
+          allowFutureAssesments: true
```

Comment:
This prop applies only to periodic assessments, yet you add it to an instant assessment. You also forgot to add it to the test dictionary.